### PR TITLE
Starting work on DepartmentLayout and SeriesLayout

### PR DESCRIPTION
**Pull request by amardesi**

### CODE REVIEW

File: app/jsx/layouts/SeriesLayout.jsx [hunk 1]
```diff
@@ -13,14 +13,43 @@ import Form from 'react-router-form'
...
   render() {
+    let p = this.props
     return (
-      <div className="o-input__droplist1">
-        <label htmlFor="c-sort1">Select</label>
-        <select name="" id="c-sort1" value={this.props.unit.id} onChange={(e)=>{browserHistory.push("/uc/"+e.target.value)}}>
-        { this.props.data.series.map((s) => 
-          <option key={s.unit_id} value={s.unit_id}>{s.name}</option>)}
-        </select>
+      <div className="c-campusselector">
```
Comment (amardesi) [inline app/jsx/layouts/SeriesLayout.jsx]: Justin has requested that this selector look just like CampusSelectorComp, but I didn't want to take the time to modify (and rename) CampusSelectorComp to be able to work with a slightly different list. So there's some duplicated code here. Would be nice to refactor someday.

File: app/jsx/layouts/DepartmentLayout.jsx [hunk 2]
```diff
@@ -19,20 +22,21 @@ class SeriesComp extends React.Component {
...
-      })).isRequired,
-      name: PropTypes.string.isRequired,
-      unit_id: PropTypes.string.isRequired
+      })).isRequired
     })
   }
   
   render() {
+    let data = this.props.data,
+        plural = (data.count == data.previewLimit + 1) ? '' : 's'
```
Comment (martinhaye) [inline app/jsx/layouts/DepartmentLayout.jsx]: Fancy

File: app/jsx/layouts/SeriesLayout.jsx (hunk 1 above)
Comment (martinhaye) [inline app/jsx/layouts/SeriesLayout.jsx]: You made the right choice.

File: app/jsx/layouts/UnitStaticPageLayout.jsx [hunk 3]
```diff
@@ -28,11 +28,11 @@ class UnitStaticPageLayout extends React.Component
           </section>
         </main>
         <aside>
-          {this.props.sidebar}
+          {this.props.sidebar.props.data.length > 1 && this.props.sidebar}
```
Comment (martinhaye) [inline app/jsx/layouts/UnitStaticPageLayout.jsx]: This looks really weird. Are you sure about 'props' being in there twice?

File: app/jsx/layouts/DepartmentLayout.jsx [hunk 4]
```diff
@@ -110,10 +131,7 @@ class DepartmentLayout extends React.Component {
             </section>
           </main>
           <aside>
-            <section className="o-columnbox1">
-              <ShareComp type="unit" id={this.props.unit.id} />
-            </section>
-            {this.props.sidebar}
+            {this.props.sidebar.props.data.length > 1 && this.props.sidebar}
```
Comment (martinhaye) [inline app/jsx/layouts/DepartmentLayout.jsx]: I don't think this is working. I know for instance that uclalaw has a sidebar, and it's not showing up now.

**Pull request merged**